### PR TITLE
Move session path to a more agnostic place (that exists!!)

### PR DESCRIPTION
**Pull request by febbraro**
From a bug Critter reported last week

### CODE REVIEW

File: root/etc/confd/templates/www.conf.tmpl [hunk 1]
```diff
@@ -217,7 +217,6 @@ catch_workers_output = yes
 ;                specified at startup with the -d argument
 ;php_admin_value[sendmail_path] = /usr/sbin/sendmail -t -i -f www@my.domain.com
 php_flag[display_errors] = off
-;php_admin_value[error_log] = /opt/rh/php55/root/var/log/php-fpm/www-error.log
```
Comment (grayside) [inline root/etc/confd/templates/www.conf.tmpl:220]: This right here... if we could get this to work with a similarly generic path, we could move the stdout symlink to base as well and not manage it as a PHP-version variation. That's aside the point of this PR, just excited to see it.

File: Dockerfile [hunk 2]
```diff
@@ -7,6 +7,8 @@ RUN yum -y install \
 
 EXPOSE 80
 
+RUN mkdir -p /var/lib/php/session
```
Comment (grayside) [inline Dockerfile:10]: Should this be a volume as well for session resume after a container restart?

Comment (tekante) [reply Dockerfile:10]: Wouldn't that be a project/environment level decision to map a volume to this particular path?

Comment (grayside) [reply Dockerfile:10]: I'm speaking of the Dockerfile VOLUME syntax, which persists data across docker runs but not docker rm.

File: root/etc/confd/templates/www.conf.tmpl (hunk 1 above)
Comment (febbraro) [reply root/etc/confd/templates/www.conf.tmpl:220]: there is no reason we can't put that right into `/var/log/php`. Let's get this merged and I will review the various branches in `apache-php` and we can consolidate.

File: Dockerfile (hunk 2 above)
Comment (febbraro) [reply Dockerfile:10]: http://stackoverflow.com/a/40174052

**Pull request merged**